sources/packages/backend/libs/services/src/disbursement-schedule/disbursement-schedule-shared.service.ts

```diff
@@ -444,11 +444,11 @@ export class DisbursementScheduleSharedService extends RecordDataModelService<Di
     applicationNumber: string,
     entityManager: EntityManager,
   ): Promise<Record<string, number>> {
-    // Get already sent (disbursed) values to know the amount that the student already received.
+    // Get already ready to send or sent (disbursed) values to know the amount that the student already received.
     const disbursementSchedules = await this.getDisbursementsForOverawards(
       studentId,
       applicationNumber,
-      DisbursementScheduleStatus.Sent,
+      [DisbursementScheduleStatus.ReadToSend, DisbursementScheduleStatus.Sent],
       entityManager,
     );
 
```

Comment:
Shouldn't we consider `ReadToSend` ones for the disbursementSchedules? (line 460)